projects/qwt.cmake

```diff
@@ -30,15 +30,15 @@ function(download_qwt)
     return()
   endif()
 
-  xpNewDownload(${PRO_QWT})
+  ipDownload(${PRO_QWT})
 endfunction()
 ########################################
 function(patch_qwt)
   if(NOT (XP_DEFAULT OR XP_PRO_QWT))
     return()
   endif()
 
-  xpPatch(${PRO_QWT})
+  ipPatch(${PRO_QWT})
```

Comment:
I'll submit a pull request to get rid of all the `ip` function calls... they should only be used internally by externpro https://github.com/smanders/externpro/blob/18.01.1/modules/xpfunmac.cmake#L4 and there shouldn't be any reason for overriding the mkpatch, download, and patch steps any more